assets/src/block-validation/sidebar.js

```diff
@@ -144,7 +183,7 @@ export function Sidebar() {
 						) ) }
 					</ul>
 				)
-					: saved && (
+					: ! isDraft && (
 						<PanelBody opened={ true }>
 							<p>
 								{ __( 'All AMP validation issues have been reviewed.', 'amp' ) }
```

Comment:
How is this condition reached? I couldn't seem to get this to show up. I only can see either a list of errors (reviewed or not) or the message that says, “There are no AMP validation issues.”

Reply:
There was no way to reach that. I've noticed that we've been [automatically falling back to displaying the reviewed errors if there were no unreviewed ones](https://github.com/ampproject/amp-wp/blob/b6dfc6a767aa18ae2389a247d01e2d8ca1ff34ae/assets/src/block-validation/sidebar.js#L62-L65).

This is now addressed in bba488b (along with some code cleanup around).